.github/workflows/publish-helm-chart.yaml

```diff
@@ -0,0 +1,41 @@
+name: Publish Helm chart
```

Comment:
Can we include a number in the workflow name so it shows up at the right place in the workflow list - (maybe 4 or 5?)

Reply:
I think 5 is better because this is not included in release workflow of the SDK.  
But I cannot find about 5 in https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/252#issuecomment-1482732796 . Can I use 5?

Reply:
I gave number 5 to Helm chart in new commit. Let me know if it has problem(s).